lib/Doctrine/DBAL/Schema/AbstractAsset.php

```diff
@@ -24,7 +24,7 @@
 abstract class AbstractAsset
 {
     /** @var string */
-    protected $_name;
+    protected $_name = '';
```

Comment:
Since it comes from a newer major release, I'd double check if it can cause any BC break. Do such changes belong to a patch release at all or the support for a new backwards-incompatible PHP version deserves a minor release?

Reply:
All code that I could find that accesses this property already assumes that it is a string. In most cases, the value is used in a string concatenation operation which would implicitly cast it to an empty string.